sdks/python/apache_beam/testing/util.py

```diff
@@ -80,9 +125,11 @@ def assert_that(actual, matcher, label='assert_that'):
 
   Args:
     actual: A PCollection.
-    matcher: A matcher function taking as argument the actual value of a
-      materialized PCollection. The matcher validates this actual value against
-      expectations and raises BeamAssertException if they are not met.
+    matcher: Two options:
+      1. A matcher function taking as argument the actual value of a
+      materialized PCollection. The matcher validates this actual value
+      against expectations and raises BeamAssertException if they are not met.
+      2. An instance of WindowedValueMatcher.
```

Comment:
Rather than branching here an introducing a new WindowedValueMatcher, perhaps introduce a windowed_equal_to sibling to equal_to.

Reply:
I thought of doing that, but how does assert_that know if matcher accepts values or TestWindowedValues?